app/views/beta/search_results/show.html.erb

```diff
@@ -1,3 +1,12 @@
+<% content_for :top_breadcrumbs do %>
+  <%= generate_breadcrumbs(
+    t('breadcrumb.search_results'),
+    [
+      [t('breadcrumb.home'), home_path]
+    ]
+  ) %>
+<% end %>
+
 <h1>Search results for '<%= @search_result.search_query_parser_result.corrected_search_query %>'</h1>
```

Comment:
I think we should use:
```
<%= page_header "Search results for
                &lsquo;#{h @search_result.search_query_parser_result.corrected_search_query}&rsquo;".html_safe
%>
```

this way includes  the switch button too:
<img width="1062" alt="Screenshot 2022-10-18 at 12 08 10" src="https://user-images.githubusercontent.com/58971/196436533-408c1d3f-2e22-4fad-a75b-e7fae2d1933a.png">

Reply:
here is how I implemented it, in case you want to reuse some parts: https://github.com/trade-tariff/trade-tariff-frontend/pull/1103/files#diff-87a700faf2c560811479f70fb39aa863eb17d5dd6072198a4520a0f5cd3cd5d9R1-R3

Reply:
Well, I guess when we merge yours we'll get that so don't need to do it here?